lib/manager/bundler/artifacts.ts

```diff
@@ -85,10 +86,21 @@ export async function updateArtifacts(
     logger.debug('No Gemfile.lock found');
     return null;
   }
+
+  if (config.isLockFileMaintenance) {
+    await fs.remove(lockFileName);
```

Comment:
Instead of importing `fs-extra`, could you write a new function in https://github.com/bilby91/renovate/blob/add-bundler-lock-file-maintenance/lib/util/fs.ts called `deleteLocalFile()` and use that instead?

Reply:
Done.